### ISSUE TITLE
FAQ for occam-ra 

### ISSUE THREAD

**Issue by venkatachalapathy** [occam-ra/occam#20]
If  we hope to attract data science, ML or Stat folks, we need to provide some resources for interested contributors.  In the absence of well written documentation or tutorial, I suggest that we use FAQ format to answer some of the questions people might have. For example, 

* Why yet another ML package? 
* If deep learning can do everything, why do we need this package? 
* Isn't `scikit-learn` good enough? 
* What is so great about `occam-ra` and graphical models? 
* What is the relationship between RA and other graphical models? 
* There are plenty of options for me in CRAN, why pick this? 
* Who has used this package? 
*  Have they used this in industry?

**Comment by gdcutting** [comment]
Yes, agreed. There is an existing FAQ at pdx.edu but I don't know that much of it is relevant for here. It's more in the way of things that someone might encounter after they've used OCCAM, whereas it would be helpful to have more info to help someone understand what it is and how to get started using it.

This overlaps with a couple of issues at the .io repo (particularly the getting-started material and tutorial content):
https://github.com/occam-ra/occam-ra.github.io/issues
so keep that in mind.

**Comment by gdcutting** [comment]
I haven't forgotten about this, but haven't had a chance to make much progress on it yet. Perhaps we can discuss this more on Friday (Mar. 22 work session)?

**Comment by venkatachalapathy** [comment]
Got it. Since we are mobilizing around our meetings, should we __close__ this issue and refer to the docs_and_guidelines channel after creating a occamFAQ.md?
